usr/share/rear/layout/save/default/445_guess_bootloader.sh

```diff
@@ -83,7 +85,9 @@ for block_device in /sys/block/* ; do
         continue
     fi
     # Check the default cases of known bootloaders:
-    for known_bootloader in GRUB2-EFI EFI GRUB2 GRUB ELILO LILO ; do
+    # cf. https://github.com/rear/rear/issues/2137
+    # s390 - zipl boot loader for rhel, ubuntu
+    for known_bootloader in GRUB2-EFI EFI GRUB2 GRUB ELILO LILa zIPL ; do
```

Comment:
This must be
```
    for known_bootloader in GRUB2-EFI EFI GRUB2 GRUB ELILO LILO zIPL ; do
```
i.e. `LILO` instead of `LILa`

Reply:
fixed